modules/oe_newsroom_newsletter/src/Form/SubscribeForm.php

```diff
@@ -102,7 +95,7 @@ public function buildForm(array $form, FormStateInterface $form_state, array $di
       '#type' => 'container',
     ];
     $form['intro_text']['content'] = [
-      '#markup' => $intro_text,
+      '#plain_text' => $intro_text,
```

Comment:
```suggestion
      '#type' => 'processed_text',
      '#text' => $intro_text,
```
I think this is more correct for rendering a textarea field, otherwise, it is rendering all text inlined.